app/scripts/proactive/model/Task.js

```diff
@@ -68,10 +68,14 @@ define(
                         "data-help":"<li><b>Name</b>: Name of the variable</li><li><b>Value</b>: Value of the variable</li><li><b>Inherited</b>: Job Variable&#39;s value will be used</li>"},
                     itemToString: Utils.inlineNameValueInherited, itemTemplate: Utils.bigCrossTemplate,
                     subSchema: {
-                    "Name": { validators: ['required'], fieldAttrs: {'placeholder': '@attributes->name'}, title: 'Name', type: 'Text', editorClass: 'popup-input-text-field' },
+                    "Name": { validators: ['required'], fieldAttrs: {'placeholder': '@attributes->name', 'title': 'Name of the variable. It must start with  a letter, can contain only alphanumeric and - , _ or . characters.'}, title: 'Name', type: 'Text', editorClass: 'popup-input-text-field' },
```

Comment:
Name of the variable. It must start with a letter and it can contain only alphanumeric and "-", "_" or "." characters.